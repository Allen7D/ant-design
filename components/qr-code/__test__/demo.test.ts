import demoTest from '../../../tests/demo';

demoTest('QrCode');
