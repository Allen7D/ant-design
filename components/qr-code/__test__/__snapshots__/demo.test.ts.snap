// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QrCode /> renders download.tsx correctly 1`] = `
<DocumentFragment>
  <div
    id="myqrcode"
  >
    <div
      class="tech-qrcode"
      style="width: 128px; height: 128px;"
    >
      <canvas
        height="128"
        ke="http://www.baidu.com"
        style="height: 128px; width: 128px;"
        width="128"
      />
    </div>
    <button
      class="ant-btn ant-btn-default"
      style="margin-block-start: 16px;"
      type="button"
    >
      <span>
        下载二维码
      </span>
    </button>
  </div>
</DocumentFragment>
`;

exports[`<QrCode /> renders errorlevel.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders logo.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders outdated.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders popover.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders refresh.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders rendermode.tsx correctly 1`] = `<DocumentFragment />`;

exports[`<QrCode /> renders withoutlogo.tsx correctly 1`] = `<DocumentFragment />`;
